# Configuration of the Rviz Interface.

# General info.
#Header header

#Maximum distance error allowed arround the objective
float32 max_error

